core/src/network/reducers/get.rs

```diff
@@ -0,0 +1,436 @@
+use crate::{
```

Comment:
This file and all the functions etc. in here should also be `query` instead of `get`